compiler/rustc_codegen_ssa/src/target_features.rs

```diff
@@ -36,6 +36,7 @@ const ARM_ALLOWED_FEATURES: &[(&str, Option<Symbol>)] = &[
     // #[target_feature].
     ("thumb-mode", Some(sym::arm_target_feature)),
     ("thumb2", Some(sym::arm_target_feature)),
+    ("d32", Some(sym::arm_target_feature)),
```

Comment:
Hm, given that these three features seem to be fated to remain unstable, maybe a different feature gate would be warranted for these?

Reply:
The 32-bit ARM features are bit of a mess and will need to be reworked anyways before stabilization can be considered.